src/planner_test.cc

```diff
@@ -831,6 +831,40 @@ TEST(PlannerTest, DetectOutRangeAcceleration) {
   VerifyCommonExpectations(plantest.segments(), *plantest.GetConfig());
 }
 
+// If we start from zero speed and enqueue a line of segments
+// at constat max feedrate we expect the profile to always be symmetrical.
+TEST(PlannerTest, StraightLine_ProfileSymmetry) {
+  MachineControlConfig *config = new MachineControlConfig();
+  InitTestConfig(config);
+  // The acceleration should stay high enough to avoid making
+  // to avoid numerical issues due to square root calculations
+  // for which the acceleration and deceleration, if they
+  // require too many steps, will end up being very inaccurate.
+  PlannerHarness plantest(0, 0, config);
+  unsigned kNumSteps = 200;
+  AxesRegister pos = {};
+
+  for (unsigned i = 0; i < kNumSteps; ++i) {
+    pos[AXIS_X] += 10;
+    plantest.Enqueue(pos, 10000);
```

Comment:
10000 also looks like a constant that might be good if named.

Reply:
(solved)